### PR TITLE
Add Typeform promotion card

### PR DESCRIPTION
**Pull request by exu3**
This PR adds the Typeform promotion card in the Perks tab as well as the task card on the admin dashboard.

Screen Shot 2021-09-24 at 2 02 18 AM

### CODE REVIEW

File: app/views/events/promotions.html.erb [hunk 1]
```diff
@@ -174,6 +174,19 @@
...
+<div class="card mb3">
+ <div class="card__banner card__banner--top flex justify-between items-center" style="background-image:url('https://cloud-m6r7s83e4-hack-club-bot.vercel.app/0blue-bg-2.png'); background-size:100% auto; background-repeat:no-repeat; padding:20px;">
+    <%= image_tag 'typeform.svg', width: 180 %>
+    <%= link_to 'Apply',
+      organizer_signed_in? ? "https://hack.af/typeform?prefill_Project%20Name=#{@event.name}&prefill_Bank%20URL=#{event_url(@event)}&prefill_Your%20name=#{current_user.full_name}" : root_path,
+      class: "btn bg-success mr1 #{'disabled' unless organizer_signed_in?}"
+    %>
+  </div>
+  <p class="mb0">
+    Typeform is partnering with Hack Club Bank to offer <strong>30% off</strong> any paid plan to organizers of all Hack Club Bank projects. Typeform is a web-based tool used to create unique online forms and surveys.
```
Comment (garyhtou) [inline app/views/events/promotions.html.erb]: Link to Typeform so they can learn more about it

File: app/views/events/promotions.html.erb [hunk 2]
```diff
@@ -174,6 +174,19 @@
   </p>
 </div>
 
+<div class="card mb3">
+ <div class="card__banner card__banner--top flex justify-between items-center" style="background-image:url('https://cloud-m6r7s83e4-hack-club-bot.vercel.app/0blue-bg-2.png'); background-size:100% auto; background-repeat:no-repeat; padding:20px;">
```
Comment (garyhtou) [inline app/views/events/promotions.html.erb]: Can you check the mobile view?
image

**Review by garyhtou — CHANGES_REQUESTED** [review]
Amazing job!

**Comment by exu3** [comment]
Fixed! thanks Gary :D

**Comment by garyhtou** [comment]
Looks good now! Thanks!
image

give me a ping once the contract and everything has been 👍 by Zach and I'll merge

**Review by garyhtou — APPROVED** [review]

**Comment by MelSmith104** [comment]
hey just reviewed this - it looks great. changed the background and the logo on the application form. hold off on merging until we get confirmation from zach - we're looking to see if they can increase their discount. also please move typefom down below sendy as it's not free

**Comment by exu3** [comment]
marking this as a draft for now, awaiting confirmation from Typeform

**Comment by maxwofford** [comment]
This looks like it's on pause so I'm closing it for now. Feel free to reopen or continue commenting here when there are updates.

**Pull request closed without merging**